pkg/retry/retry.go

```diff
@@ -141,52 +146,61 @@ func (s *Server) handleRetry() http.Handler {
 				// append to the total failure events counter
 				failedEventCount += 1
 
-				logger.Infow("redeliver failed event", "eventID", *event.ID, "guid", *event.GUID)
-
-				if err := s.github.RedeliverEvent(ctx, *event.ID); err != nil {
-					var acceptedErr *github.AcceptedError
-					if errors.As(err, &acceptedErr) {
-						logger.Infow("skipping redeliver event because it has already been submitted to GitHub",
-							"eventID", *event.ID,
-							"guid", *event.GUID,
-							"error", err,
-						)
-						continue
-					}
-
-					// found an unaccepted error, check if its already in the events table
-					exists, err := s.datastore.DeliveryEventExists(ctx, s.eventsTableID, *event.GUID)
-					if err != nil {
-						logger.Errorw("failed to call BigQuery",
-							"method", "DeliveryEventExists",
-							"code", http.StatusInternalServerError,
-							"body", errDeliveryEventExists,
-							"error", err,
-						)
-						http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
-						return
-					}
-
-					if exists {
-						logger.Infow("skipping redeliver event because it has already been processed", "eventID", *event.ID, "guid", *event.GUID)
-						continue
-					}
-
-					logger.Errorw("failed to call RedeliverEvent, stop processing",
+				failedEventsHistory = append(failedEventsHistory, &EventIdentifier{eventID: event.ID, guid: *event.GUID})
```

Comment:
Is `event` a pointer?
```suggestion
				failedEventsHistory = append(failedEventsHistory, &EventIdentifier{eventID: event.ID, guid: event.GUID})
```

Reply:
it is a pointer of this following type `*github.HookDelivery`